fern/definition/medical/patient.yml

```diff
@@ -196,8 +189,11 @@ types:
       address:
         type: optional<Addresses>
       contact:
-        type: optional<list<Contact>>
-
+        type: optional<Contacts>
```

Comment:
@jonahkaye can you try `optional<list<Contact>>` and lets delete the `Contacts` type?